hq/test/logic/InspectorResultsTest.scala

```diff
@@ -12,6 +11,8 @@ import scala.collection.JavaConverters._
 
 
 class InspectorResultsTest extends FreeSpec with Matchers with OptionValues {
+  import logic.InspectorResults._
```

Comment:
This fixes a weird compile issue. Also included in https://github.com/guardian/security-hq/pull/140 so we'll have a minor conflict when these are both merged whichever order we do them in.